### ISSUE TITLE
Spanish translations inconsistency

### ISSUE THREAD

**Issue by Sergih28** [Felienne/hedy#623]
**There isn't a clear way to address to the user in the [Spanish translation](https://github.com/Felienne/hedy/blob/main/coursedata/level-defaults/es.yaml).**

Sometimes it addresses to the user in second person (which is the **normal way** in Spanish), and sometimes does it in third person (which is the **formal way**).

Here are some examples:

- puede**s**. Using _second person_. Line 3:
```
¡Bienvenido a Hedy! En el Nivel 1 puedes usar los comandos `print`,` ask` y `echo`.
```

- **su**. Using _third person_. Line 5:
```
En el lado izquierdo de su pantalla están todos los comandos del Nivel 1.
```

- Presion**e**. Using _third person_. Line 7:
```
¡Presione el botón azul y el código será escrito por ti!
```

- Prueb**a**. Using _second person_. Line 9:
```
Prueba el código tu mismo con el botón verde 'Ejecutar el código' debajo del campo de programación de la izquierda.
```

---

Both are correct, but think **we should stick to one**.
I would suggest to go with the _second person_ form, because it makes sense for what Hedy is.

**Comment by fpereiro** [comment]
Hi @Sergih28 ! That is indeed right - we should use the second person form to be consistent with the style of Hedy. I'll do that on all upcoming translations, and at some later point review the existing ones.

**Comment by Sergih28** [comment]
I can help reviewing the current ones if you want @fpereiro

**Comment by fpereiro** [comment]
> I can help reviewing the current ones if you want @fpereiro

That would be fantastic! Thank you!

**Comment by fpereiro** [comment]
Marking this as closed after merging #629 . @Sergih28 feel free to reopen it if you think something's missing!
